lib/virtual-cards/spending-limit.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { VirtualCardLimitInterval } from '../graphql/types/v2/graphql';
+
+export const getLimitIntervalString = spendingLimitInterval => {
```

Comment:
@Betree what do you think of this regarding localization practices?

Reply:
Personally not a fan of returning React components from `lib,` but I have no real argument to support that except that it's more verbose.

The real issue is that what's done here is inconsistent with what we do elsewhere: export a function from `lib/i18n/___.js`.

Prefixing with `/` also seems out of scope for this function.

Reply:
I refactored to keep this code under `/lib/i18n/virtual-card-spending-limit.ts`.

Wrt to returning a react node vs string, it depends. 

`intl.formatMessage` can return a react node natively if there is formatting like `<Strong>{virtualCardName}</Strong>`. Considering this, using a function from `/lib/i18n` does not guarantee it will return a pure string unless we are enforcing it  as a best practice or in some other way.

I found this to be useful here, I used a single i18n format string to be able to display the available balance and format the individual parts. I can break it into different intl strings if necessary.
https://github.com/opencollective/opencollective-frontend/pull/9097/files#diff-f0e02682411335da9a903e2825aa454477524258884f188e51b889c744b37d1aR65
![image](https://github.com/opencollective/opencollective-frontend/assets/5448927/5c6a731f-04c7-4769-abff-60f5fa81e0b7)

Reply:
That makes sense. Most of the helpers in the same folders are not dealing with formatting, but if that's required here I don't see it as a blocker.